pages/api/getuserlist.js

```diff
@@ -0,0 +1,24 @@
+
+
+import dbConnect from '../../utils/dbConnect'
+import User from '../../models/User'
+import Friends from '../../models/Friends'
+
+export default async (req, res) => {
+  const { name, email } = req.query
+
+  if (( !name || !email )) {
+    return res.status(422).json({ message: 'Invalid input' })
+  }
+
+  await dbConnect()
+
+  try {
+    const requester = await User.findOne({ name: name, email: email })
+    const users = await User.find({})
+    const friends = await Friends.find({})
```

Comment:
is this WIP?

Reply:
I use it in the line right after.
`res.status(200).json({ success: true, users: users, friends: friends })`